superset/views/core.py

```diff
@@ -2503,10 +2503,13 @@ def sql_json(self):
             return json_error_response(payload=data)
         # SORU Modifications
         tab_id = data['query']['tab']
-        superset_id = re.search('<@(.+?)>', tab_id).group(1)
-        redis_conn = redis.StrictRedis.from_url('redis://redis:6379/0')
-        key = 'superset_{}'.format(superset_id)
-        redis_conn.set(key, payload, 300)
+        if tab_id:
```

Comment:
[] wouldn't really return None if the key does not exist. You'll need to use get(). However, in this particular case it might just be better/more pythonic if you could use
```
try:
  ...
except Exception as e
  log.debug or warn
  pass
```

Reply:
The key exists in that case. It is just null. I don't want to cover all exceptions in here. This is for a specific valid case. When the preview runs this method is called with the tabid null